README.md

```diff
@@ -26,6 +26,18 @@ And open http://localhost:9000/frontend
 
 This will restart the server on any changes: shared code, client/server, assets.
 
+## Tests
```

Comment:
It'd be better to rephrase it saying that "npm install jsdom" or "yarn add jsdom" is a pre-requisite to running `sbt> test`